package.json

```diff
@@ -126,9 +128,5 @@
         }
       ]
     ]
-  },
```

Comment:
@yuhao900914 For future reference, the `dependencies` section of a package.json file is only for dependencies that are used in the actual library code that we ship to users.  Anything listed under `dependencies` will be installed by our users when they install @amplitude/react-native.  Our library does not have any dependencies, other than `react-native`, which is listed under `peerDependencies` because we don't download it for the user.

`devDependencies` = Dependencies that are only used during development/build time.  Typedoc and it's theme are good examples, they don't power the library but they are used occassionally to generate docs.  Same with `semantic-release`, we use it to run releases, but our users don't need to know about it.

`peerDependencies` - Dependencies that are required for the shipped npm library to function.  These are not installed when installing @amplitude/react-native, it is expected that the user installs these themselves.  For example don't automatically install `react-native` when installing @amplitude/react-native.

`dependencies` = Dependencies that the library requires and will be installed at the same time @amplitude/react-native is installed.  For example, if we used an npm package in index.ts to provide some piece of functionality, we would include it here.  We currently do not have any dependencies, they are all dev or peer.

Reply:
That's very clear. Thanks for pointing that out.

Reply:
Not a problem!  Glad it was helpful :). These 3 always tripped me up but building npm packages helped clear up a lot for me.